node/test/graphql/mutation/me/config-update-featured.spec.ts

```diff
@@ -0,0 +1,183 @@
+              featuredMentors
+              featuredMentorPanels
+              activeMentors
+            }
+          }
+        }`,
+        variables: {},
+      });
+    expect(response.status).to.equal(400);
+  });
+
+  it('does not accept api key user', async () => {
+    const response = await request(app)
+      .post('/graphql')
+      .set('mentor-graphql-req', 'true')
+      .set('Authorization', `bearer ${process.env.API_SECRET}`)
+      .send({
+        query: `mutation UpdateConfigFeatured($config: ConfigUpdateFeaturedInputType!) {
+          me {
+            updateConfigFeatured(config: $config) {
+              mentorsDefault
+              featuredMentors
+              featuredMentorPanels
+              activeMentors
+            }
+          }
+        }`,
+        variables: { config: {} },
+      });
+    expect(response.status).to.equal(200);
+    expect(response.body).to.have.deep.nested.property(
+      'errors[0].message',
+      'you do not have permission to edit config'
+    );
+  });
+
+  it('does not accept USER', async () => {
+    const token = getToken('5ffdf41a1ee2c62320b49ea2'); //mentor with role "User"
+    const response = await request(app)
+      .post('/graphql')
+      .set('Authorization', `bearer ${token}`)
+      .send({
+        query: `mutation UpdateConfigFeatured($config: ConfigUpdateFeaturedInputType!) {
+          me {
+            updateConfigFeatured(config: $config) {
+              mentorsDefault
+              featuredMentors
+              featuredMentorPanels
+              activeMentors
+            }
+          }
+        }`,
+        variables: { config: {} },
+      });
+    expect(response.status).to.equal(200);
+    expect(response.body.errors[0].message).to.equal(
+      'you do not have permission to edit config'
+    );
+  });
+
+  it('CONTENT_MANAGER can update config', async () => {
+    const token = getToken('5ffdf41a1ee2c62320b49ea5'); //mentor with role "Content Manager"
+    const response = await request(app)
+      .post('/graphql')
+      .set('Authorization', `bearer ${token}`)
+      .send({
+        query: `mutation UpdateConfigFeatured($config: ConfigUpdateFeaturedInputType!) {
+          me {
+            updateConfigFeatured(config: $config) {
+              mentorsDefault
+              featuredMentors
+              featuredMentorPanels
+              activeMentors
+            }
+          }
+        }`,
+        variables: {
+          config: {
+            featuredMentors: ['5ffdf41a1ee2c62111111119'],
+            featuredMentorPanels: ['5ffdf41a1ee2c62111111111'],
+          },
+        },
+      });
+    expect(response.status).to.equal(200);
+    expect(response.body.data.me.updateConfigFeatured).to.eql({
+      activeMentors: [],
+      featuredMentorPanels: ['5ffdf41a1ee2c62111111111'],
+      featuredMentors: ['5ffdf41a1ee2c62111111119'],
+      mentorsDefault: [],
+    });
+  });
+
+  it('ADMIN can update config', async () => {
```

Comment:
If you have time, tests for partial updates that prove that other fields won't be overwritten are also good